homeassistant/components/climate/__init__.py

```diff
@@ -534,6 +534,14 @@ async def async_turn_on(self) -> None:
             await self.hass.async_add_executor_job(self.turn_on)
             return
 
+        # If there are only two HAVC modes, and one of those modes is OFF,
```

Comment:
```suggestion
        # If there are only two HVAC modes, and one of those modes is OFF,
```

Reply:
It is also in the description. I would make the same error 😃